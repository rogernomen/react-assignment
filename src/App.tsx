import React from "react";
import "./App.css";
import MenuPage from "./features/menu";
import { QueryClient, QueryClientProvider } from "react-query";

function App(): React.ReactElement {
	const queryClient = new QueryClient({
		defaultOptions: {
			queries: {
				refetchOnWindowFocus: false
			}
		}
	});

	return (
		<QueryClientProvider client={queryClient}>
			<div className="App">
				<MenuPage />
			</div>
		</QueryClientProvider>
	);
}

export default App;
