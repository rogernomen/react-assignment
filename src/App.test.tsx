import React from "react";
import renderer from "react-test-renderer";
import App from "./App";

describe("the App Component", () => {
	it("should be rendered", () => {
		expect.assertions(1);
		const component = renderer.create(<App></App>);
		const tree = component.toJSON();

		expect(tree).toMatchSnapshot();
	});
});
