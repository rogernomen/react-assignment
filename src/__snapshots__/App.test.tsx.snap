// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the App Component should be rendered 1`] = `
<div
  className="App"
/>
`;
